plugins/CoreHome/EntityDuplicator/DuplicateRequestResponse.php

```diff
@@ -36,6 +38,15 @@ class DuplicateRequestResponse
      */
     protected $additionalData;
 
+    /**
+     * @var array Optional array containing the data required for the event to be posted on success. If set, the event
+     * to record the activity will be triggered when the getResponseArray method is called.
```

Comment:
The event is not specifically to record the activity, any plugin can hook into it with its functionality, so perhaps we can remove the mention of recording an activity here.